tests/lib/prepare.sh

```diff
@@ -790,6 +790,27 @@ EOF
                            --output "$IMAGE_HOME/$IMAGE"
     rm -f ./pc-kernel_*.{snap,assert} ./pc_*.{snap,assert} ./snapd_*.{snap,assert}
 
+    if is_core20_system; then
+        # (ab)use ubuntu-seed
+        LOOP_PARTITION=2
+    else
+        LOOP_PARTITION=3
+    fi
+
+    # expand the uc16 and uc18 images a little bit (400M) as it currently will
+    # run out of space easily from local spread runs if there are extra files in
+    # the project not included in the git ignore and spread ignore, etc.
+    if ! is_core20_system; then
+        # grow the image by 400M
+        truncate --size=+400M "$IMAGE_HOME/$IMAGE"
+        # fix the GPT table because old versions of parted complain about this and
+        # refuse to properly run the next command unless the GPT table is updated
+        sgdisk "$IMAGE_HOME/$IMAGE" -e
```

Comment:
FYI, `-e` is `--move-second-header` and moves the backup gpt partition to the end of the disk. This is sensible since we've just resized the backing storage. I think a comment update warranted (in a follow-up) to explain why this is sensible.

Reply:
sure I can file a followup with the no-spread tag changing this comment

Reply:
done in #8880